test/llvmpasses/Makefile

```diff
@@ -0,0 +1,11 @@
+SRCDIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
+JULIAHOME := $(abspath $(SRCDIR)/../..)
+include $(JULIAHOME)/deps/Versions.make
+include $(JULIAHOME)/Make.inc
+include $(JULIAHOME)/deps/llvm-ver.make
+BUILDDIR = $(JULIAHOME)/deps/scratch
+include $(JULIAHOME)/deps/llvm-options.mk
+LLVM_SRC_DIR:=$(JULIAHOME)/deps/srccache/llvm-$(LLVM_VER)
```

Comment:
You can build it out of tree, but you need a copy of lit which is in the LLVM source dir. You can probably make it work, but might be easier for somebody who uses out-of-tree builds to fix this up after committing this.